openshift-hack/e2e/annotate/rules.go

```diff
@@ -165,14 +165,8 @@ var (
 			// https://issues.redhat.com/browse/OCPBUGS-7297
 			`DNS HostNetwork should resolve DNS of partial qualified names for services on hostNetwork pods with dnsPolicy`,
 
-			// https://issues.redhat.com/browse/OCPBUGS-45275
-			`\[sig-network\] Connectivity Pod Lifecycle should be able to connect to other Pod from a terminating Pod`,
-
 			// https://issues.redhat.com/browse/OCPBUGS-17194
 			`\[sig-node\] ImageCredentialProvider \[Feature:KubeletCredentialProviders\] should be able to create pod with image credentials fetched from external credential provider`,
-
-			// https://issues.redhat.com/browse/OCPBUGS-45273
-			`\[sig-network\] Services should implement NodePort and HealthCheckNodePort correctly when ExternalTrafficPolicy changes`,
```

Comment:
OK to skip this test, see my technical reasoning : https://issues.redhat.com/browse/OCPBUGS-45273?focusedId=26796850&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-26796850

however this test should be inside: "[Disabled:Broken]": {
// tests that are known broken and need to be fixed upstream or in openshift knownBroken tags because we need to fix something in ocp and u/s for us to have this working properly.

Reply:
Done